applications/SEOR-code-less-smart-contract-platform.md

```diff
@@ -0,0 +1,155 @@
+
+####Overall plan
+
+We expect to implement a codeless smart contract platform that can support multiple chains through four phases.  
+* Implement some popular standard parameterized multi-chain contracts, such as ERC20, ERC721, etc. At the same time complete the basic contract deployer, multi-chain support SDK and a multi-chain DApp that allows users to operate.  
+
+* Add some standard parameterized contracts for complex businesses to the contract set, such as DEX business and liquid mining business. At the same time upgrade the supporting SDK and DApp.  
+
+* Carry out business integration within the contract collection to enable some composite businesses to form a more complete business flow, such as one-click completion: ERC20 Token issuance + Token sales + DEX listing + liquidity mining mine. And upgrade the contract deployer, multi-chain DApp and other infrastructure.  
+
+* Support more parallel chains and public chains that support smart contracts, upgrade SDK and DApp, and propose a complete standard access protocol to implement an open platform.
+
+Implementation content of this grant application:
+We will use this grant to advance the development of the first and second phases. For details, please refer to the milestone section.
+
+### Ecosystem Fit 
+No
+
+## Team :busts_in_silhouette:
+
+### Team members
+* Winnie Wen - team leader
+* Yingxuan Li - full-stack engineer
+* Huaiyu Wang - full-stack engineer
+
+### Team Website	
+* https://www.seor.io
+
+### Legal Structure 
+* Name: SEALSC FOUNDATION LIMITED
+* Address: 2 VENTURE DRIVE, #11-31, VISION EXCHANGE, SINGAPORE 608526
+* UEN: 201931689N
+
+### Team's experience
+Our team is made up of leaders, core development engineers, and community developers of leading blockchain enterprise and projects Onchain, Ontology, TRON, and Bitkeep. We have rich technical accumulation and in-depth understanding in the blockchain field. We are working on blockchain's standardize and protocolize, and will provide a complete infrastructure solution to enable blockchains to powered various fields.
+
+### Team Code Repos
+https://github.com/SealSC
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 95 days
+* **Full-time equivalent (FTE):**  3
+* **Total Costs:** 12500 DAI
+
+### Milestone 1 - Multi-chain smart contract set
+* **Estimated Duration:** 1 month
+* **FTE:**  3
+* **Costs:** 5000 DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 1. | License | Apache License 2.0 |
+| 2. | Contract set Repository | A git repository containing the source code of 4 contracts: parameterized ERC20, liquidity mining, contract deployer and Uniswap protocol connector. These smart contracts are all developed to realizes parameterized during deployment, not copy the standard contract such as ERC20 directly. All of contracts will be will be implemented using both solidity and ink! technologies. |
```

Comment:
Furthermore, are you referring to a github repository by `Contract set Repository`? So that means in the future, the community could add additional parametrised contracts, right?

Reply:
Yes, our current plan is to create a github repository under this organization: https://github.com/SealSC, but due to our initial code is not yet ready, it has not yet been actually created.  

We will accept the new contract code under this repository and handle issues, PRs and proposals from the community and any other interested third parties.